lib/cache/cache.go

```diff
@@ -1467,7 +1467,17 @@ func (c *Cache) GetRemoteCluster(clusterName string) (types.RemoteCluster, error
 		ta(cachedRemote)
 		return cachedRemote.Clone(), nil
 	}
-	return rg.presence.GetRemoteCluster(clusterName)
+	rc, err := rg.presence.GetRemoteCluster(clusterName)
+	if trace.IsNotFound(err) && rg.IsCacheRead() {
+		// release read lock early
+		rg.Release()
+		// fallback is sane because method is never used
```

Comment:
Naive question: what does "method" mean here?

Reply:
Refers to the method in which the comment appears.  Updated comment to be a little more clear.